src/content/reference/react-dom/unmountComponentAtNode.md

```diff
@@ -39,21 +39,21 @@ render(<App />, domNode);
 unmountComponentAtNode(domNode);
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lainnya di bawah ini.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `domNode`: A [DOM element.](https://developer.mozilla.org/en-US/docs/Web/API/Element) React will remove a mounted React component from this element.
+* `domNode`: Sebuah [elemen DOM.](https://developer.mozilla.org/en-US/docs/Web/API/Element) React akan menghapus komponen React yang terpasang dari elemen ini.
 
-#### Returns {/*returns*/}
+#### Pengembalian {/*returns*/}
```

Comment:
```suggestion
#### Kembalian {/*returns*/}
```
Reference: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320
I also just updated the Glossary accordingly.